runtime/service.go

```diff
@@ -612,12 +617,26 @@ func (s *service) buildVMConfiguration(req *proto.CreateVMRequest) (*firecracker
 		driveBuilder = firecracker.NewDrivesBuilder(defaultRootfsPath)
 	}
 
-	// TODO: Reserve fake drives here (https://github.com/firecracker-microvm/firecracker-containerd/pull/154)
-	// Right now, just a single hardcoded stub drive is allocated
-	driveBuilder = driveBuilder.AddDrive("/dev/null", false, func(drive *models.Drive) {
-		drive.IsRootDevice = firecracker.Bool(false)
-		drive.DriveID = firecracker.String("containerRootfs")
-	})
+	stubDriveIndex := int64(len(driveBuilder.Build()) - 1)
+	containerCount := int(req.ContainerCount)
+	if containerCount == 0 {
+		containerCount = 1
```

Comment:
A comment here would be appropriate.  Something like: The container count should always be positive so that at least one container can run inside the VM.